pom.xml

```diff
@@ -293,8 +293,8 @@
     </dependency>
     <dependency>
       <groupId>org.bouncycastle</groupId>
-      <artifactId>bcprov-jdk15on</artifactId>
-      <version>1.70</version>
+      <artifactId>bcprov-jdk18on</artifactId>
+      <version>1.75</version>
```

Comment:
```suggestion
      <version>1.76</version>
```

Reply:
@rakatyal @shrutiburman please commit this change and merge this PR to fix the CVE vulnerability

Reply:
@claudiachua as well